lib/module/stories/stories_view.dart

```diff
@@ -136,6 +144,14 @@ class HnStoriesPageState extends State<HnStoriesPage>
               _handleThemeChange(ThemeName.dark);
             },
           ),
+          const Divider(),
+          new ListTile(
+            title: const Text('Show full comments'),
+            trailing: new Checkbox(
```

Comment:
I think we could we use a https://docs.flutter.io/flutter/material/Switch-class.html instead